File: python/ut9_bases_de_datos/coches_crud/app/domain/model/coche.py
class Coche:
    def __init__(self, matricula :str, marca :str, 
                 modelo :str, color :str):
        self.matricula = matricula
        self.marca = marca
        self.modelo = modelo
        self.color = color

    def __str__(self):
        return f"Matricula: {self.matricula}, Marca: {self.marca}, Modelo: {self.modelo}, Color: {self.color}"

File: programación/ejercicios_viejos/gestionvehiculos/Coche.java
package gestionvehiculos;

public class Coche extends Vehiculo {

    private boolean esElectrico;

    public Coche(String marca, String modelo, int anioFabricacion, boolean esElectrico) {
        super(marca, modelo, anioFabricacion);
        this.esElectrico = esElectrico;
    }

    public boolean isEsElectrico() {
        return esElectrico;
    }
}
